polaris-react/src/components/DropZone/components/FileUpload/FileUpload.module.scss

```diff
@@ -14,7 +14,7 @@
 }
 
 .small {
-  padding: var(--p-space-300);
+  padding: var(--p-space-200);
```

Comment:
Curious why this change was needed?

Reply:
I have to change it in order to fix height for 40px size:

https://github.com/Shopify/polaris/assets/104942025/4fa6b255-2dd8-4980-b3c4-5b55dda23711